rules.py

```diff
@@ -912,6 +912,47 @@ def check(self):
                 "The Election File has following issues with the identifiers.", error_log)
 
 
+class AllCaps(base.BaseRule):
+    """The Name elements in Candidates, Contests and Person elements should not be in all uppercase.
```

Comment:
Please use a sample XML file to make sure it catches the types of errors you expect.